all/entrypoint.sh

```diff
@@ -71,6 +74,13 @@ while test $# -gt 0; do
             GEN_GATEWAY=true
             shift
             ;;
+        --lint)
+            LINT=true
+            if test $# -gt 1; then
```

Comment:
This is going to take the next arg no matter what - i.e. if we say `--lint --with-gateway`, it'll set `LINT_CHECKS=--with-gateway`. Also you'll need another `shift` inside the `if` block, or you'll re-process the next command.

I think what you'll want to do is say `if [[ $2 == -* ]]; then`, i.e. check that the next arg starts with a `-`.

Reply:
Ah, yes you're right. Thank you for catching that. Working on a fix now.